libs/base/pxtbase.h

```diff
@@ -815,6 +815,7 @@ struct ImageHeader {
 
 class RefImage : public RefObject {
   public:
+    uint16_t revision;
```

Comment:
I guess could just be uint8_t now that I'm thinking of it, since it only matters that it matches / if it overflows back to 0 it'd be fine unless there are exactly 256 mutations separate mutations on the sprite within a single frame

Reply:
it doesn't matter - allocations are word-aligned; probably best to make it uint32_t to be safe and avoid weird bugs

however do move it after the buffer field